be/test/storage/sstable/persistent_index_sstable_test.cpp

```diff
@@ -102,6 +104,8 @@ TEST_F(PersistentIndexSstableTest, test_generate_sst_seek_and_check) {
         IndexValue cur_val(UNALIGNED_LOAD64(iter->value().get_data()));
         ASSERT_TRUE(exp_val == cur_val);
     }
+    delete iter;
+    delete sstable;
```

Comment:
sstable should be initialized by nullptr.

Reply:
Done